proto/elys/amm/denom_liquidity.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package elysnetwork.elys.amm;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/elys-network/elys/x/amm/types";
+
+message DenomLiquidity {
+  string denom = 1; 
+  string liquidity = 2 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.moretags) = "yaml:\"taken_out_min_amount\"", (gogoproto.nullable) = false];
```

Comment:
more tag name `taken_out_min_amount` looks invalid.
Could we just use sdk.Coin object instead of DenomLiquidity struct? I think it would be better to reduce number of newly introducing structs

Reply:
good catch will fix that thanks